benchmark/scanner/scanner.py

```diff
@@ -17,6 +20,66 @@ def __init__(self, scanner_type: str, scanner_url: str, working_dir: str, cred_d
         self.true_cnt: int = 0
         self.false_cnt: int = 0
         self.result_dict: dict = {}
+        self.total_true_cnt = 0
+        self.total_false_cnt = 0
+        self.total_template_cnt = 0
+        self.categories: Dict[str, Tuple[int, int, int]] = {}  # true_cnt, false_cnt, template_cnt
+        self.next_id = 0
+        self.total_data_valid_lines = 0
+        self.meta: List[Dict[str, Any]] = []
+        self._read_meta()
+
+    def _read_meta(self):
+        for root, dirs, files in os.walk(f"{self.cred_data_dir}/meta"):
+            for file in files:
+                with open(f"{root}/{file}", newline="") as f:
+                    reader = csv.DictReader(f)
+                    for row in reader:
+                        if not row["Category"]:
+                            # all unmarked categories are Other
+                            row["Category"] = "Other"
+                        if row["Category"] not in self.categories:
+                            # init the counters
+                            self.categories[row["Category"]] = (0, 0, 0)
+                        true_cnt, false_cnt, template_cnt = self.categories[row["Category"]]
+                        if row["GroundTruth"] == "T":
+                            true_cnt += 1
+                            self.total_true_cnt += 1
+                        elif row["GroundTruth"] == "F":
+                            self.total_false_cnt += 1
+                            false_cnt += 1
+                        elif row["GroundTruth"] == "Template":
+                            self.total_template_cnt += 1
+                            template_cnt += 1
+                        else:
+                            # wrong markup should be detected
+                            print(row, flush=True)
```

Comment:
What about add some string which make recognize the row is marked up wrong more easily?

```suggestion
                            # wrong markup should be detected
                            print(f"[WRONG MARKUP] {row}", flush=True)
```